### PR TITLE
feat: add `ai` service with `deepgram` and `openai`

### PR DESCRIPTION
**Pull request by altaywtf**
feat: add DEEPGRAM and OPENAI keys to env

feat: init transcribeAudio with deepgram

feat: add openai

feat: add summarizeTranscript via OPENAI

refactor: use `@env` to access openai key

feat: add generateEpisodeContent

### CODE REVIEW

**Comment by altaywtf** [comment]
Current dependencies on/for this PR:
* main
  * **PR #12** <a href="https://app.graphite.dev/github/pr/experiment-station/beecast/12?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #14** <a href="https://app.graphite.dev/github/pr/experiment-station/beecast/14?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #17** <a href="https://app.graphite.dev/github/pr/experiment-station/beecast/17?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/experiment-station/beecast/17?utm_source=stack-comment).

**Comment by altaywtf** [comment]
@okanisildar updated this to be more granular

File: lib/services/ai/generate-episode-content.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      .from('episode_content')
+      .upsert(
+        {
+          episode: episodeId,
+          text_summary: summary,
+          transcription: transcript,
+          user: await getAccountId(),
+        },
+        {
+          onConflict: 'episode',
```
Comment (altaywtf) [inline lib/services/ai/generate-episode-content.ts:52]: @okanisildar added this 👍

**Review by okanisildar — APPROVED** [review]

**Comment by altaywtf** [comment]
### Merge activity

* **Dec 13, 10:00 AM**: @@altaywtf merged this pull request with [Graphite](https://app.graphite.dev/github/pr/experiment-station/beecast/17).

**Pull request merged**